packages/common/icons/src/utils.tsx

```diff
@@ -0,0 +1,17 @@
+export enum Network {
```

Comment:
I'm not sure this is a good idea...we should try to actually fix the import issue.

Reply:
It was being a pain in the ass and didn't seem worth it to me for one function and one enum